src/components/layout/Shell.tsx

```diff
@@ -110,9 +110,9 @@ export function Shell(props: Props) {
           className="py-4 md:py-0"
         >
           <Box>
-            <Link to="/admin/products">
+            <LinkWithSearchParams href="/admin/products">
```

Comment:
for wouter`s `Link` `href` and `to` are the same thing. One is an alias for another.